tree/tree/src/TBranchElement.cxx

```diff
@@ -2845,11 +2845,19 @@ TString TBranchElement::GetFullName() const
       // The parent's name is already included in the name for split TClonesArray and STL collections
       return fName;
    }
-   TString motherName(mother->GetName());
-   if (motherName.Length() && (motherName[motherName.Length()-1] == '.')) {
+
```

Comment:
Can this just call `TBranch::GetFullName` at this point?

Reply:
Even: Replace it by TBranch::GetFullName.
I propose to remove the override

Reply:
> I propose to remove the override

Done now.